src/components/filters/filter_menu/filter_menu.ts

```diff
@@ -53,13 +42,6 @@ const CSS = css/* scss */ `
       display: flex;
       flex-direction: row;
       margin-bottom: 4px;
-
-      .o-filter-menu-action-text {
-        cursor: pointer;
-        margin-right: 10px;
-        color: blue;
-        text-decoration: underline;
-      }
     }
 
     .o-filter-menu-list {
```

Comment:
All the borders of the side panel have the same gray IIRC (GRAY_300 I think ?). Maybe this one should too